tests/pcovr.py

```diff
@@ -0,0 +1,139 @@
+import unittest
```

Comment:
We need documentation for tests, otherwise, it's not easy to figure out what they should do at all.